sources/packages/backend/workflow/test/test-utils/mock/zeebe-worker-mock.ts

```diff
@@ -0,0 +1,138 @@
+import { AssessmentConsolidatedData } from "../../models";
+import { Duration, ZBClient, ZBWorkerConfig } from "zeebe-node";
+import { ApplicationExceptionStatus, ProgramInfoStatus } from "@sims/sims-db";
+import { Workers } from "../constants/worker-constants";
+
+/**
+ * Variables provided for the purpose of e2e tests
+ * when creating an instance of assessment gateway workflow.
+ */
+interface FakeAssessmentVariables {
+  e2eTestStudentStatus: string;
+  e2eTestApplicationExceptionStatus: ApplicationExceptionStatus;
+  e2ePIRStatus: ProgramInfoStatus;
+}
+
+const fakeWorkers: ZBWorkerConfig<FakeAssessmentVariables, unknown, unknown>[] =
+  [
+    {
+      taskType: Workers.AssociateWorkflowInstance,
+      taskHandler: (job) =>
+        job.complete({ [Workers.AssociateWorkflowInstance]: true }),
+    },
+    {
+      taskType: Workers.SaveDisbursementSchedules,
+      taskHandler: (job) =>
+        job.complete({ [Workers.SaveDisbursementSchedules]: true }),
+    },
+    {
+      taskType: Workers.SaveAssessmentData,
+      taskHandler: (job) =>
+        job.complete({ [Workers.SaveAssessmentData]: true }),
+    },
+    {
+      taskType: Workers.UpdateNOAStatus,
+      taskHandler: (job) => job.complete({ [Workers.UpdateNOAStatus]: true }),
+    },
+    {
+      taskType: Workers.LoadAssessmentConsolidatedData,
+      taskHandler: (job) =>
+        job.complete(getMockConsolidatedData(job.variables)),
```

Comment:
In general, what if we allow the assessment workflow to receive the completed mocked object to be returned by the worker?
For instance, if `load-assessment-consolidated-data` is supposed to return:
```json
{
    "someVariable1": 123,
    "someVariable2": 444,
    "someVariableN": 555,
}
```
We can start the workflow with the payload as:
```json
{
    "assessmentId": 1,
    "e2eLoadAssessmentConsolidatedDataResult": {
        "someVariable1": 123,
        "someVariable2": 444,
        "someVariableN": 555,
    }
}
```
The work from the worker method would be just to return the workflow variable `e2eLoadAssessmentConsolidatedDataResult`, thoughts?